examples/data-sources/datadog_action_connection/data-source.tf

```diff
@@ -0,0 +1,3 @@
+data "datadog_action_connection" "my_connection" {
+  id = "1d05af79-db0e-4eca-b3b2-98c4a673c8cf"
```

Comment:
nit: is this an existing id? it might be better to use a dummy id like `11111111-2222-3333-4444-555555555555`

Reply:
This repo dismisses all reviews if I make any changes so I'd rather leave this alone :grimacing: